src/google/protobuf/descriptor.cc

```diff
@@ -7675,6 +7676,22 @@ bool DescriptorBuilder::OptionInterpreter::ExamineIfOptionIsSet(
   return true;
 }
 
+template <typename T> std::string ValueOutOfRange(
+    std::string type_name, std::string option_name) {
+  return absl::StrFormat(
+    "Value out of range, %d to %d, for %s option \"%s\".", \
+    std::numeric_limits<T>::min(), std::numeric_limits<T>::max(),
+    type_name, option_name);
+}
+
+template <typename T> std::string ValueMustBeInt(
+    std::string type_name, std::string option_name) {
```

Comment:
please pass both of these as `absl::string_view`

Reply:
Done